src/Core/Managed/Shared/Extensibility/Implementation/Tracing/PortalDiagnosticsSender.cs

```diff
@@ -87,13 +87,7 @@ public void Send(TraceEvent eventData)
         }
 
         private void InternalSendTraceTelemetry(TraceEvent eventData)
-        {
-            // If channel is not initialized yet, do nothing
-            if (this.telemetryClient.Channel == null)
```

Comment:
Its a required check. Added a internal Getter for the config in the TelemetryClient, and this is now bought back.